examples/ordering/src/network.rs

```diff
@@ -3,19 +3,22 @@ use aleph_bft::{NodeIndex, Recipient};
 use aleph_bft_mock::{Hasher64, PartialMultisignature, Signature};
 use codec::{Decode, Encode};
 use log::error;
-use std::{io::Write, net::SocketAddr};
+use std::net::SocketAddr;
 use tokio::{
-    io::{self, AsyncReadExt},
-    net::TcpListener,
+    io,
+    net::UdpSocket,
     time::{sleep, Duration},
 };
 
+const MAX_UDP_DATAGRAM_BYTES: usize = 65536;
+
 pub type NetworkData = aleph_bft::NetworkData<Hasher64, Data, Signature, PartialMultisignature>;
 
 pub struct Network {
     my_id: usize,
     addresses: Vec<SocketAddr>,
-    listener: TcpListener,
+    socket: UdpSocket,
+    buffer: Box<[u8; MAX_UDP_DATAGRAM_BYTES]>,
```

Comment:
Hmm, why do we need this `Box`? 
I see that we need it, as when I remove it, the thing compiles but then crashes, but why :P ?

Reply:
I think what happens in that case is that there is a lot of state (65kb) for the futures to track in their state machine, so it blows up the stack quickly. When it's allocated on the heap, that's not a problem.

Reply:
Another option would be to use a `Vec`, but I wanted to make the size explicit in the type so that there is never any error where the `Vec` is too small by accident.

Reply:
Added a short comment with explanation.